portfolio/src/main/webapp/script.js

```diff
@@ -55,19 +55,42 @@ function createList(text){
     return liElement;
 }
 
+/* This function is called upon clicking submit limit. It will set the limit for the max
+number of comments and save it in the session storage. */
 function loadComments(){
-    fetch('/load').then(response => response.json()).then((arrayString) =>
+    numberComments = document.getElementById('max').value; 
+    sessionStorage.setItem("limit",numberComments);
+
+}
+
+/*This function is called when refreshing the page. It fetches the comments up to the max amount
+determined by the max stored in session storage.*/
+function loadAllComments(){
+    const url = "/load?numComments="+sessionStorage.getItem("limit");
+    fetch(url).then(response => response.json()).then((arrayString) =>
     {
         const commentElement = document.getElementById('comment-container');
         var olTag = document.createElement('ol');
         commentElement.innerHTML = '';
         var i; 
         for (i =0; i< arrayString.length; i++){
-            olTag.appendChild(createList(arrayString[i]));
+            olTag.appendChild(createList(arrayString[i].comment));
         }
+        
         commentElement.appendChild(olTag);
+       
     });
 
 }
 
+//This function deletes all comments.
+async function deleteComments(){
+  await fetch('/delete-data', {method: 'POST'});
+  fetch('/load').then(response => response.json()).then((arrayString) =>
```

Comment:
best to be consistent, if you're going to use async await, use it for all the promises in the function